tendrl/node_agent/message/handler.py

```diff
@@ -0,0 +1,53 @@
+from gevent import socket
+from gevent.socket import error as socket_error
+from gevent.socket import timeout as socket_timeout
+import gevent.event
+import gevent.greenlet
+from gevent.server import StreamServer
+from io import BlockingIOError
+import os
+from tendrl.commons.message import Message
+from tendrl.node_agent.message.logger import Logger
+
+
+RECEIVE_DATA_SIZE = 4096
+
+class MessageHandler(gevent.greenlet.Greenlet):
+    def __init__(self):
+        super(MessageHandler, self).__init__()
+        self.server = StreamServer(
+            self.bind_unix_listener(),
+            self.read_socket
+        )
+
+    def read_socket(self, sock, address):
+        try:
+            data = sock.recv(RECEIVE_DATA_SIZE)
+            message = Message.from_json(data)
+            Logger(message)
+        except (TypeError, ValueError, KeyError,
```

Comment:
whats the point of these try except blocks ? if you are just raising the exception. This comment holds good for all the below try except blocls